tests/Omnisend/SDK/V1/ContactTest.php

```diff
@@ -10,7 +10,7 @@ final class ContactTest extends TestCase
     public function test_validation(): void
     {
         // Test valid contact
-        $contact_data = ['email' => 'test@example.com'];
+        $contact_data = ['email' => '@example.com'];
```

Comment:
does the test failed before? If not  the test itself is not working.

Reply:
Yes, the test is not working. Was trying to test it & unintentionally committed that

Reply:
If test is not working either fix it or remove at all.